templates/topic/index.json-ld

```diff
@@ -28,19 +29,195 @@
+		]
+	},
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Topic page",
+				"fr": "Page de sujet"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Topic pages provide access to topics and destination pages that support task completion.",
+				"fr": "Les pages de sujet fournissent un accès aux sujets et aux pages de destination qui appuient l’achèvement des tâches."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/mandatory-templates/theme-topic.html",
+				"fr": "https://conception.canada.ca/modeles-obligatoire/theme-sujet.html"
+			},
+			"iteration": "_:iteration_topic_3",
+			"example": [
+				{
+					"en": { "href": "topic-en.html", "text": "Topic page" },
+					"fr": { "href": "topic-fr.html", "text": "Page de sujet" }
+				}
+			],
+			"implementation": [
+				"_:implement_topic",
+				"_:implement_topic_aem"
+			],
+			"history": [
+				{
+					"en": "June 2024 - Stabilization of the version 2 of the page template.",
+					"fr": "Juin 2024 - Stabilisation de la version 2 du gabarit de page."
+				},
+				{
+					"en": "January 2021 - Provisional implementation of the version 2 of the page template.",
+					"fr": "Janvier 2021 - Implémentation provisoire de la version 2 du gabarit de page."
+				},
+				{
+					"en": "August 2014 - Implementation of the version 1 of the page template.",
+					"fr": "Août 2014 - Implémentation de la version 1 du gabarit de page."
+				}
+			]
+		}
+	],
+	"implementation": [
+		{
+			"@id": "_:implement_topic",
+			"iteration": "_:iteration_topic_3",
+			"name": {
+				"en": "Standard",
+				"fr": "Standard"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers/publishers adding the template manually.",
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le gabarit manuellement."
+			},
+			"instructions": {
+				"en": [
+					"Refer to the working example for a code sample.",
+					"Refer to the working example and guidance for more information on how to implement this page template."
+				],
+				"fr": [
+					"Reportez-vous à l'exemple pratique pour un exemple de code.",
+					"Référez-vous à l'exemple pratique et aux directives pour plus d'informations sur l'implémentation ce modèle de page."
+				]
+			}
+		},
+		{
+			"@id": "_:implement_topic_aem",
+			"iteration": "_:iteration_topic_3",
+			"name": {
+				"en": "AEM users",
+				"fr": "Utilisateurs AEM"
+			},
+			"introduction": {
+				"en": "This implementation is meant for publishers adding the template manually into an AEM page.",
+				"fr": "Cette implémentation est destinée aux éditeurs qui ajoutent le gabarit manuellement à une page d'AEM."
+			},
+			"instructions": {
+				"en": [
+					"TODO"
+				],
+				"fr": [
+					"TODO"
+				]
+			},
+			"ajaxSourceCode": {
+				"en": "topic-en.html main > *:not(.pagedetails)",
+				"fr": "topic-fr.html main > *:not(.pagedetails)"
```

Comment:
I suggest to move that code sample for the "Standard" while we are waiting for the instruction for AEM.